etcd/etcd.go

```diff
@@ -197,9 +197,9 @@ func (c *client) RegisterConfigCallback(ctx context.Context, key string, uniqueI
 			}
 		}
 	}()
-	ctx, cancel = context.WithTimeout(context.Background(), c.etcdTimeout)
```

Comment:
Better we put line 171 inside the goroutine.

Reply:
Done